src/status_im/ui/screens/navigation.cljs

```diff
@@ -16,11 +16,13 @@
 
 (fx/defn navigate-to-clean
   [{:keys [db]} view-id screen-params]
-  (log/debug "current view-id " (:view-id db)
-             "changing to " view-id)
+  (log/debug "current view-id" (:view-id db)
+             "changing to" view-id)
   (let [db (cond-> db
              (seq screen-params)
              (assoc-in [:navigation/screen-params view-id] screen-params))]
+    (when (= :login view-id)
+      (log/debug (:push-notifications/stored db)))
```

Comment:
I imagine this is temp code? If not, we should add more context here as to what we're actually logging.

Reply:
yes i will remove it, currently i'm trying to debug app